decidim-elections/app/commands/decidim/elections/admin/setup_election.rb

```diff
@@ -176,6 +178,13 @@ def notify_trustee_about_election
 
           Decidim::EventsManager.publish(data)
         end
+
+        def election_status(bb_status)
```

Comment:
I'd use an action for the name of this method, something like `update_election_bb_status` or `store_bulletin_board_status`.

Reply:
Yes I agree, it was named `set_election_status` but Rubocop was offended 🤖. 
I'll name it `store_bulletin_board_status`. Thanks 😀